build/bootstrap/bootstrap-debian.sh

```diff
@@ -70,3 +72,25 @@ sudo mv /tmp/bazelisk /usr/bin/bazel
 
 # Install the Unison file-syncer.
 . bootstrap/bootstrap-unison.sh
+
+# Configure environment variables for CockroachDB and Managed Service.
+echo 'export PATH="${PATH}:$HOME/go/src/github.com/cockroachdb/cockroach/bin:/usr/local/go/bin"' >> ~/.bashrc_bootstrap
+echo 'export PATH="${PATH}:$HOME/go/src/github.com/cockroachlabs/managed-service/bin"' >> ~/.bashrc_bootstrap
+. ~/.bashrc_bootstrap
+
+# If DISABLE_COCKROACHDB is false, clone CockroachDB.
+if [ "${DISABLE_COCKROACHDB:-false}" = "false" ]; then
+  git clone https://github.com/cockroachdb/cockroach "$(go env GOPATH)/src/github.com/cockroachdb/cockroach"
+  git -C "$(go env GOPATH)/src/github.com/cockroachdb/cockroach" submodule update --init
+fi
+
+# If DISABLE_MANAGED_SERVICE is false, clone Managed Service
+if [ "${DISABLE_MANAGED_SERVICE:-false}" = "false" ]; then
+  git clone git@github.com:cockroachlabs/managed-service.git "$(go env GOPATH)/src/github.com/cockroachlabs/managed-service"
+  git -C "$(go env GOPATH)/src/github.com/cockroachlabs/managed-service" submodule update --init
```

Comment:
This repo appears to have no submodules. Am I missing something?

Reply:
correcting it.